nimble/host/src/ble_hs_startup.c

```diff
@@ -307,7 +307,7 @@ ble_hs_startup_set_evmask_tx(void)
                                                 BLE_HCI_OCF_CB_SET_EVENT_MASK2),
                                      buf, sizeof(buf));
     if (rc != 0) {
-        return rc;
+        BLE_HS_LOG(WARN, "ble_hs_startup_set_evmask_tx() failed\n");
```

Comment:
why need this change?

Reply:
ok, good point.  I'll investigate making such a patch.

Reply:
Oh, I see #93 is the patch.  How soon can this be merged?